### DISCUSSION TITLE
Placeholder (BOTH)

### DISCUSSION THREAD

**Discussion by Erissos** [TownyAdvanced/Towny#6683] [Q&A] [answered]
![image](https://github.com/TownyAdvanced/Towny/assets/88965431/f36234c5-f2e6-4fee-aebc-f90b13895685)
I want to make the chat as in the photo but I can't find a placeholder for "BOTH". I want to collect the city and country name in one [], [German|City] if the person is not in a country, I want it to look like this: [City].
 [German] [City] looks very bad in chat.

**Comment by LlmDl** [comment, marked as answer]
https://github.com/TownyAdvanced/Towny/wiki/Placeholders
```
%townyadvanced_towny_tag% - Displays town and nation tags.

%townyadvanced_towny_tag_override% - Displays town and nation tags if they exist, falling back to names if they don't.

%townyadvanced_towny_tag_formatted% - Displays town and nation tags if they exist, falling back to long-form names if they don't.
```
